ui.apps/src/main/content/jcr_root/apps/venia/components/commerce/productcarousel/v1/productcarousel/productcarousel.html

```diff
@@ -0,0 +1,25 @@
+
+<sly data-sly-use.clientlib="/libs/granite/sightly/templates/clientlib.html" 
+data-sly-call="${clientlib.all @ categories='aem.venia.productcarousel'}"/>
+
+<sly data-sly-test="${!properties.product}">
+  <div class="blank-placeholder product-carousel-title" data-sly-text="${component.title}"></div>
+</sly>
+
+<sly data-sly-test="${properties.product}">
+  <div data-comp-is="productcarousel" class="product-carousel carousel-root">
+    <div class ="product-carousel-parent" data-sly-use.productList="${'com.adobe.cq.commerce.core.components.models.productcarousel.ProductCarousel' @productSkuList=properties.product}">
+      <div class="cards-container"  data-sly-list.product=${productList.products}>
+        <div role="displaycard" class="card product-card">
```

Comment:
It would be convenient to put the product list item rendering in a separate HTL to facilitate the customized rendering of product items.

Reply:
product rendering is moved to separate HTL file.